core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -16,41 +16,11 @@
 
 package com.google.cloud.sql.core;
 
-import com.google.api.client.http.HttpRequestInitializer;
 import com.google.cloud.sql.ConnectionConfig;
-import com.google.cloud.sql.CredentialFactory;
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Preconditions;
-import com.google.common.base.Strings;
-import com.google.common.util.concurrent.ListenableFuture;
-import com.google.common.util.concurrent.ListeningScheduledExecutorService;
-import com.google.common.util.concurrent.MoreExecutors;
-import java.io.File;
-import java.io.IOException;
-import java.net.InetSocketAddress;
-import java.net.Socket;
-import java.security.KeyPair;
-import java.security.KeyPairGenerator;
-import java.security.NoSuchAlgorithmException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import java.util.logging.Logger;
-import javax.net.ssl.SSLSocket;
-import jnr.unixsocket.UnixSocketAddress;
-import jnr.unixsocket.UnixSocketChannel;
 
 /**
- * Factory responsible for obtaining an ephemeral certificate, if necessary, and establishing a
- * secure connecting to a Cloud SQL instance.
- *
- * <p>This class should not be used directly, but only through the JDBC driver specific {@code
- * SocketFactory} implementations.
- *
- * <p>The API of this class is subject to change without notice.
+ * Implementation of informally used java API to preserve compatibility with older code that uses
```

Comment:
Will we be marking this as a deprecated class in a separate PR?

Reply:
I have now annotated this class with `@Deprecated`.